api/nameguard/web_api.py

```diff
@@ -497,14 +499,16 @@ async def secure_primary_name_get(address: str, network_name: NetworkName) -> Se
     2. For ENS names using CCIP-Read: requests to externally defined gateway servers.
 
     Returns `display_name` to be shown to users and estimates `impersonation_status`.
+
+    If `return_nameguard_report` is `False`, the `nameguard_result` field will be `None`.
```

Comment:
Please see my other comment about how we document this field in the TS library and apply similar ideas here. Thanks.